setup.cfg

```diff
@@ -1,13 +0,0 @@
-[flake8]
```

Comment:
I don't think you can remove this file. Copy the one in salobj instead:

https://github.com/lsst-ts/ts_salobj/blob/develop/setup.cfg